pkg/volume/item_snapshot.go

```diff
@@ -54,4 +54,7 @@ type ItemSnapshotStatus struct {
 
 	// Phase is the current state of the ItemSnapshot.
 	Phase isv1.SnapshotPhase `json:"phase,omitempty"`
+
```

Comment:
Need to modify line 45's json tag format
`ResourceIdentifier string "json:resourceIdentifier"`